src/Pim/Bundle/DataGridBundle/Resources/public/js/datagrid/cell/boolean-label-cell.js

```diff
@@ -18,8 +18,8 @@ define(['oro/datagrid/string-cell', 'oro/translator'],
                     return this;
                 }
 
-                var status = 'true' === value ? 'success' : 'important';
-                var label = 'true' === value ? 'Yes' : 'No';
+                var status = (true === value || '1' === value) ? 'success' : 'important';
```

Comment:
I don't remember why there was simple quote before, but I think you break the association products datagrid if you remove them